### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by kdobmayer**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+type config struct {
+	Platform string `env:"platform,opt[ios,android,'ios,android']"`
+	Readd    bool   `env:"readd_platform,opt[true,false]"`
+	Version  string `env:"cordova_version"`
+	WorkDir  string `env:"workdir,dir"`
+}
+
+func runCommand(name string, arg ...string) error {
+	log.Infof("$ %s %s", name, strings.Join(arg, " "))
```
Comment (trapacska) [inline main.go]: please do not use logging outside of main()

File: main.go [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+	cmd := exec.Command(name, arg...)
```
Comment (trapacska) [inline main.go]: we have `command.NewWithXyz` call so you can skip setting STDs, also you can just simply return with that.
And if you function is one liner, do you need the function separated?

File: main.go [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+	cmd.Stderr = os.Stderr
+	return cmd.Run()
+}
+
+func failf(format string, v ...interface{}) {
+	log.Errorf(format, v...)
+	os.Exit(1)
+}
+
+func update(version string) error {
```
Comment (trapacska) [inline main.go]:
```
if version == "latest" {
return runCommand("npm", "install", "-g", "cordova")
}else{
return runCommand("npm", "install", "-g", "cordova@"+version)
}
```

or 

```
return runCommand("npm", "install", "-g", map[bool]string{true:"cordova",false:"cordova@"+version}[version=="latest"]
```

File: main.go [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+		}
+	}
+
+	out, err := exec.Command("cordova", "-v").Output()
+	if err != nil {
+		failf("Failed to get cordova version: %v", err)
+	}
+	fmt.Printf("using cordova version %s\n", out)
+
+	if err := os.Chdir(cfg.WorkDir); err != nil {
```
Comment (trapacska) [inline main.go]: avoid using os.Chdir. Set the workDir for the command itself please

**Review by trapacska — CHANGES_REQUESTED** [review]

**Review by trapacska — APPROVED** [review]

**Review by godrei — APPROVED** [review]

**Pull request merged**